python/pipeline/meso.py

```diff
@@ -1267,19 +1308,25 @@ def make(self, key):
         masks = Segmentation.reshape_masks(pixels, weights, image_height, image_width)
 
         # Classify masks
-        if key['classification_method'] == 1:  # manual
-            if not SummaryImages() & key:
-                msg = 'Need to populate SummaryImages before manual mask classification'
-                raise PipelineException(msg)
-
-            template = (SummaryImages.Correlation() & key).fetch1('correlation_image')
-            masks = masks.transpose([2, 0, 1])  # num_masks, image_height, image_width
-            mask_types = mask_classification.classify_manual(masks, template)
-        elif key['classification_method'] == 2:  # cnn-caiman
-            from .utils import caiman_interface as cmn
-            soma_diameter = tuple(14 / (ScanInfo.Field() & key).microns_per_pixel)
-            probs = cmn.classify_masks(masks, soma_diameter)
-            mask_types = ['soma' if prob > 0.75 else 'artifact' for prob in probs]
+        print('Currently only implemented for classification of manually segmented glomeruli masks.')
```

Comment:
I placed this print statement here because I removed the code to classify the `soma` compartment, which is in the `cajal/pipeline`.  I didn't keep the `soma` classification within this set of if statements because `soma` classification isn't used for this project, many compartments were labeled `unknown`, and it would have been a bigger rework.  Does this make sense?

Reply:
As discussed, will keep this print statement.